app/grandchallenge/verifications/signals.py

```diff
@@ -93,3 +100,22 @@ def auto_disable_user_in_verification_user_set(
                         vus_link=verification_user_set.get_absolute_url(),
                     ),
                 )
+
+
+@receiver(post_save, sender=Verification)
```

Comment:
Signals should only be used when absolutely necessary. There is no reason to use a signal here. This should be done in the save method of the verification, and only be invoked if `is_verified` has changed to `True`, see usages of `FieldChangeMixin` to help you with that.